From df56e07a75453bd20ebb8f6e4c5dd3b74cf3a484 Mon Sep 17 00:00:00 2001
From: Rongwei Wang <rongwei.wang@linux.alibaba.com>
Date: Thu, 30 Apr 2020 10:52:20 +0800
Subject: [PATCH v3 devel-4.19.91-008 3/3] alios:arm64:config: add msr config

fix #25760036

add CONFIG_ARM64_MSR config, and set CONFIG_ARM64_MSR=m defaultly.

Signed-off-by: Rongwei Wang <rongwei.wang@linux.alibaba.com>
Reviewed-by: Zou Cao <zoucao@linux.alibaba.com>
---
 configs/kernel-4.19-aarch64-alios7.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/kernel-4.19-aarch64-alios7.config b/configs/kernel-4.19-aarch64-alios7.config
index 05c3779..04f3c5d 100644
--- a/configs/kernel-4.19-aarch64-alios7.config
+++ b/configs/kernel-4.19-aarch64-alios7.config
@@ -4809,6 +4809,7 @@ CONFIG_QCOM_L3_PMU=y
 CONFIG_XGENE_PMU=y
 CONFIG_ARM_SPE_PMU=y
 CONFIG_RAS=y
+CONFIG_ARM64_MSR=m
 
 #
 # Android
-- 
1.8.3.1

